### PR TITLE
Demo that creates a map from generated key/values

### PR DESCRIPTION
**Pull request by mhutchinson**
The output map tiles are simply summarized into a single local file

### CODE REVIEW

**Comment by mhutchinson** [comment]
This now generates one file per tile for simplicity, while preserving all of the information (the initial commit in this PR threw away lots of information to summarize the tiles into a single file).

There is a verifier provided which can confirm inclusion of values by verifying the merkle structure.

I think this is a minimal enough demo that shows how this could be worked into a solution and the value it provides.

File: experimental/batchmap/cmd/build/mapdemo.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	beam.RegisterType(reflect.TypeOf((*writeTileFn)(nil)).Elem())
+}
+
+func main() {
+	flag.Parse()
+	beam.Init()
+
+	output := filepath.Clean(*output)
+	if output == "" {
+		log.Fatal("No output provided")
```
Comment (AlCutter) [inline experimental/batchmap/cmd/build/mapdemo.go:62]: Could use `glog` for consistency with the verifier below, and then switch to `Exitf` in here?

File: experimental/batchmap/cmd/verify/verify.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	key          = flag.Int64("key", 0, "This is the seed for the key that will be looked up.")
+	prefixStrata = flag.Int("prefix_strata", 1, "The number of strata of 8-bit strata before the final strata.")
+)
+
+func main() {
+	flag.Parse()
+
+	mapDir := filepath.Clean(*mapDir)
+	if mapDir == "" {
+		glog.Fatal("No output provided")
```
Comment (AlCutter) [inline experimental/batchmap/cmd/verify/verify.go:50]: Can use `Exitf`

**Review by AlCutter — APPROVED** [review]
nice!

Couple nitty suggestions inline, but LGTM

File: experimental/batchmap/cmd/build/mapdemo.go (hunk 1 above)
Comment (mhutchinson) [reply experimental/batchmap/cmd/build/mapdemo.go]: Cool, done.

File: experimental/batchmap/cmd/verify/verify.go (hunk 2 above)
Comment (mhutchinson) [reply experimental/batchmap/cmd/verify/verify.go:50]: Done.

**Pull request merged**